partner_code_program_territory/tests/test_partner_code_program_territory.py

```diff
@@ -0,0 +1,94 @@
+from odoo.tests.common import Form, TransactionCase
+
+
+class TestPartnerCodeProgramTerritory(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
2025-06-30 11:23:32,290 250 WARNING odoo py.warnings: /__w/sy-partner-contact/sy-partner-contact/partner_code_program_territory/tests/test_partner_code_program_territory.py:1: DeprecationWarning: Since 18.0: odoo.tests.common.Form is deprecated, use odoo.tests.Form